src/getVariants.ts

```diff
@@ -1,26 +1,27 @@
-import { AtRule, Node } from 'postcss';
+import postcss, { AtRule, Node } from 'postcss';
+import postcssSelectorParser from 'postcss-selector-parser';
 
 import { JitState } from './types';
 
 function isAtRule(node: Node): node is AtRule {
   return node.type === 'atrule';
 }
 
-export function getVariants(state: JitState): Record<string, string | null> {
-  function escape(className: string): string {
-    const node = state.modules.postcssSelectorParser.module.className({ value: className });
-    return node.value;
-  }
+function escape(className: string): string {
+  const node = postcssSelectorParser.className({ value: className });
+  return node.value;
+}
 
+export function getVariants(state: JitState): Record<string, string | null> {
```

Comment:
what do you think of just passing variantMap and separator to getVariants?
seems even cleaner

Reply:
I passed `jitContext` instead, as both `variantMap` and `separator` can be derived from that.